csrc/multidevice/ipc_handle.cpp

```diff
@@ -247,7 +247,10 @@ SymmetricMemoryHandle* SymmetricMemoryHandleCache::get(KeyType key) {
   // If not found, create a new handle based on the expr type
   std::unique_ptr<SymmetricMemoryHandle> handle;
 
-  if (auto* comm = dynamic_cast<Communication*>(key.expr)) {
+  if (auto* dtca = dynamic_cast<hir::SymmetricContiguousView*>(key.expr)) {
```

Comment:
Can you replace `dtca` with a better name? It's unclear to me what that stands for.